packages/explorer/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@latticexyz/explorer",
+  "version": "2.1.0",
+  "private": true,
+  "description": "Explorer is a tool for visually exploring and manipulating the state of worlds",
+  "bin": {
+    "explorer": ".next/standalone/packages/explorer/server.js"
+  },
+  "scripts": {
+    "build": "next build && cp -r .next/static .next/standalone/packages/explorer/.next && pnpm link .",
+    "dev": "next dev",
+    "lint": "next lint",
+    "start": "node .next/standalone/packages/explorer/server.js"
+  },
+  "dependencies": {
+    "@hookform/resolvers": "^3.9.0",
+    "@latticexyz/common": "workspace:*",
+    "@latticexyz/protocol-parser": "workspace:*",
+    "@latticexyz/schema-type": "workspace:*",
+    "@latticexyz/store": "workspace:*",
+    "@latticexyz/world": "workspace:*",
+    "@radix-ui/react-checkbox": "^1.1.1",
+    "@radix-ui/react-dropdown-menu": "^2.1.1",
+    "@radix-ui/react-label": "^2.1.0",
+    "@radix-ui/react-select": "^2.1.1",
+    "@radix-ui/react-separator": "^1.1.0",
+    "@radix-ui/react-slot": "^1.1.0",
+    "@radix-ui/react-toast": "^1.2.1",
+    "@radix-ui/themes": "^3.0.5",
+    "@tanstack/react-query": "^5.51.3",
+    "@tanstack/react-table": "^8.19.3",
+    "@uidotdev/usehooks": "^2.4.1",
+    "@wagmi/core": "^2.12.1",
+    "better-sqlite3": "^8.6.0",
+    "class-variance-authority": "^0.7.0",
+    "clsx": "^2.1.1",
+    "lucide-react": "^0.408.0",
+    "next": "14.2.3",
+    "next-themes": "^0.3.0",
+    "query-string": "^9.1.0",
+    "react": "^18",
+    "react-contenteditable": "^3.3.7",
+    "react-dom": "^18",
+    "react-highlight-words": "^0.20.0",
+    "react-hook-form": "^7.52.1",
+    "sonner": "^1.5.0",
+    "tailwind-merge": "^1.12.0",
+    "tailwindcss-animate": "^1.0.7",
+    "viem": "2.9.20",
+    "wagmi": "^2.11.1",
+    "zod": "3.23.8",
+    "zustand": "^4.3.7"
+  },
+  "devDependencies": {
+    "@trivago/prettier-plugin-sort-imports": "^4.3.0",
+    "@types/better-sqlite3": "^7.6.4",
+    "@types/node": "^20",
+    "@types/react": "^18",
+    "@types/react-dom": "^18",
+    "@types/react-highlight-words": "^0.20.0",
+    "eslint": "^8",
```

Comment:
same with this, this is already in the workspace root

Reply:
removed, couldn't spot any more